sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -14,7 +14,9 @@ import { TokenCredential } from '@azure/core-auth';
 export type AnalyzeSentimentErrorResult = TextAnalyticsErrorResult;
 
 // @public
-export type AnalyzeSentimentOptions = TextAnalyticsOperationOptions;
+export interface AnalyzeSentimentOptions extends TextAnalyticsOperationOptions {
+    includeOpinionMining?: boolean;
```

Comment:
Am I dreaming or wasn't this called `mineOpinions` before?

Reply:
It was and I was told to change it to `includeOpinionMining`: https://github.com/Azure/azure-sdk-for-js/pull/10484#discussion_r467085271.